Cargo.toml

```diff
@@ -15,20 +15,21 @@ http = "0.2"
 hyper = { version = "0.14", default-features = false, features = ["client"] }
 log = { version = "0.4.4", optional = true }
 rustls-native-certs = { version = "0.6", optional = true }
-rustls = { version = "0.21.6", default-features = false }
+rustls = { version = "0.22.0-alpha.3", default-features = false }
```

Comment:
Should use `=` (same for the other alpha versions).

Reply:
Again, for all the alpha versions.

Reply:
Done